src/validation.h

```diff
@@ -385,9 +385,6 @@ bool TestBlockValidity(CValidationState& state, const CChainParams& chainparams,
 /** Check whether witness commitments are required for block. */
 bool IsWitnessEnabled(const CBlockIndex* pindexPrev, const Consensus::Params& params);
 
-/** Check whether NULLDUMMY (BIP 147) has activated. */
```

Comment:
*sigh* Probably should update the comment for `IsWitnessEnabled` above...

Might be good to note that segwit is enforced for all transactions, and this function is only used for nulldummy and block-level things (ie, working out if you've upgraded and need to reprocess segwit blocks that you didn't download witness data for, if you should generate witness data for getblocktemplate, and so you can accept old blocks from non-witness peers)

Reply:
comment updated